packages/rs-dpp/src/data_contract/accessors/mod.rs

```diff
@@ -36,21 +36,21 @@ impl DataContractV0Getters for DataContract {
         }
     }
 
-    fn document_type(&self, name: &str) -> Result<DocumentTypeRef, ProtocolError> {
+    fn document_type_for_name(&self, name: &str) -> Result<DocumentTypeRef, ProtocolError> {
```

Comment:
It's not how people call methods in Rust. I will send you my conversation with ChatGPT. He suggested me those names.